tcmalloc/deallocation_profiler.cc

```diff
@@ -309,9 +309,10 @@ class DeallocationProfiler {
   std::unique_ptr<DeallocationStackTraceTable> reports_ = nullptr;
 
  public:
-  explicit DeallocationProfiler(DeallocationProfilerList* list) : list_(list) {
+  explicit DeallocationProfiler(DeallocationProfilerList* list, bool seed_with_live_allocs = true):
```

Comment:
Do you need to modify many more additional callsites if you remove this default?
It would be clearer to explicitly specify this flag.